shared/src/business/useCases/public/getPublicDownloadPolicyUrlInteractor.test.js

```diff
@@ -0,0 +1,52 @@
+const {
+  getPublicDownloadPolicyUrlInteractor,
+} = require('./getPublicDownloadPolicyUrlInteractor');
+const { Document } = require('../../entities/Document');
+const { MOCK_CASE } = require('../../../test/mockCase');
+
+describe('getPublicDownloadPolicyUrlInteractor', () => {
+  beforeEach(() => {});
+
+  it('should throw an error for a document that is not public accessible', async () => {
+    const applicationContext = {
+      getPersistenceGateway: () => ({
+        getCaseByCaseId: () => ({ ...MOCK_CASE }),
+        getPublicDownloadPolicyUrl: () => 'localhost',
+      }),
+    };
+    await expect(
+      getPublicDownloadPolicyUrlInteractor({
+        applicationContext,
+        caseId: '123',
+        documentId: 'c6b81f4d-1e47-423a-8caf-6d2fdc3d3859',
+      }),
+    ).rejects.toThrow();
+  });
+
+  it('should return a url for a document that is public accessible', async () => {
```

Comment:
After the interactor gets the case and document based on the passed params it will determine if the document is publicly accessible. THEN it calls `getPublicDownloadPolicyUrl` to return the URL - which should be a _REAL_ URL.

In this particular test we're mocking that out, (see line 30). I'm going to update this so it looks more truthful.